autopush/router/simple.py

```diff
@@ -61,9 +61,15 @@ def amend_endpoint_response(self, response, router_data):
         """Stubbed out for this router"""
 
     def stored_response(self, notification):
+        self.metrics.gauge("notification.message_data",
```

Comment:
True, but I wanted to get these calls about as close to the point that the action occurs as possible. Everything goes through `route_notification`, and that's really the only time we can say with authority what the disposition is & have the data size. 

I'll add the calls to the other methods as well. (Although bridge would pretty much only be "Direct")

... and just realized there's a bunch of metrics we missed for the audit.